theme_paptic/static/src/scss/primary_variables.scss

```diff
@@ -84,6 +84,6 @@ $o-color-palettes-compatibility-indexes: (
 // Shapes
 //------------------------------------------------------------------------------
 
-$o-bg-shapes: change-shape-colors-mapping('web_editor', 'Blocks/04', (3: 1));
+$o-bg-shapes: change-shape-colors-mapping('web_editor', 'Blocks/04', (3: 1, 1: 2));
```

Comment:
Should this be done in this commit? I don't see Blocks/04 changed in the community PR, is this a fix? If yes, it should be done in another PR.

Reply:
The render of this snippet is not the same than the screenshot (paptic_screenshot.jpg).
We'll probably update that in another PR with some other updates on this theme :). I removed the [FIX] commit.